satpy/tests/writer_tests/test_cf.py

```diff
@@ -270,15 +270,22 @@ def test_single_time_value(self):
         start_time = datetime(2018, 5, 30, 10, 0)
         end_time = datetime(2018, 5, 30, 10, 15)
         test_array = np.array([[1, 2], [3, 4]])
+        test_array2 = test_array[..., np.newaxis]
         scn['test-array'] = xr.DataArray(test_array,
                                          dims=['x', 'y'],
                                          coords={'time': np.datetime64('2018-05-30T10:05:00')},
                                          attrs=dict(start_time=start_time,
                                                     end_time=end_time))
+        scn['test-array2'] = xr.DataArray(test_array2,
+                                          dims=['x', 'y', 'time'],
+                                          coords={'time': [np.datetime64('2018-05-30T10:05:00')]},
```

Comment:
What happens if `test-array2` has a different time coordinate?

Reply:
If I change the value of from `np.datetime64('2018-05-30T10:05:00')` to `np.datetime64('2018-05-30T10:10:00')` for `test-array2`, it would raise this error for `scn.save_datasets()`:
```
E                   ValueError: arguments without labels along dimension 'time' cannot be aligned because they have different dimension size(s) {1} than the size of the aligned dimension labels: 2
```

I suppose they should have the same time coordinate, right? Otherwise, it should be MultiScene.

Reply:
Wasn't this the case with Himawari where the channels had slightly different timestamps?

Reply:
Ha, then this sounds difficult now. The `time_bounds` variable has the time dimension which is different from the dataset:

`time_bounds`:
```
<xarray.DataArray (time: 1, bnds_1d: 2)>
array([['2018-05-30T10:00:00.000000000', '2018-05-30T10:15:00.000000000']],
      dtype='datetime64[ns]')
Dimensions without coordinates: time, bnds_1d
```

`dataset`:
```
<xarray.Dataset>
Dimensions:      (time: 2, x: 2, y: 2)
Coordinates:
  * time         (time) datetime64[ns] 2018-05-30T10:05:00 2018-05-30T10:10:00
Dimensions without coordinates: x, y
Data variables:
    test-array   (time, x, y) float64 1.0 2.0 3.0 4.0 nan nan nan nan
    test-array2  (x, y, time) float64 nan 1.0 nan 2.0 nan 3.0 nan 4.0
```

Reply:
How about renaming `time` to `time_1d` for `time_bounds`?

Reply:
I tested the Himawari-8 data:

```
multi_files = glob('../data/H8/HS_H08_20210601_00[0-1]0_B0*_FLDK_R10_S0110.DAT.bz2')
mscn = MultiScene.from_files(multi_files, reader='ahi_hsd')
blended_scene = mscn.blend(blend_function=timeseries)
blended_scene.save_datasets(filename='test_multiscn.nc', writer='cf')
```

Yes. I got this error above. Then, I checked the `time_bounds` and `dataset`:

```
time_bounds:

<xarray.DataArray (time: 2, bnds_1d: 2)>
array([['2021-06-01T00:00:20.954731000', '2021-06-01T00:00:54.559133000'],
       ['2021-06-01T00:10:20.945215000', '2021-06-01T00:10:54.549616000']],
      dtype='datetime64[ns]')
Dimensions without coordinates: time, bnds_1d

dataset:

<xarray.Dataset>
Dimensions:         (time: 4, x: 11000, y: 11000)
Coordinates:
  * time            (time) datetime64[ns] 2021-06-01T00:00:20.954731 ... 2021...
  * y               (y) float64 5.5e+06 5.499e+06 ... -5.499e+06 -5.5e+06
  * x               (x) float64 -5.5e+06 -5.499e+06 ... 5.499e+06 5.5e+06
    B01_start_time  (time) datetime64[ns] 2021-06-01T00:00:20.954731 NaT ... NaT
    B01_end_time    (time) datetime64[ns] 2021-06-01T00:00:54.310009 NaT ... NaT
    longitude       (y, x) float64 dask.array<chunksize=(1100, 4096), meta=np.ndarray>
    latitude        (y, x) float64 dask.array<chunksize=(1100, 4096), meta=np.ndarray>
    B02_start_time  (time) datetime64[ns] NaT ... 2021-06-01T00:10:21.194340
    B02_end_time    (time) datetime64[ns] NaT ... 2021-06-01T00:10:54.549616
Data variables:
    FLDK            int64 0
    B01             (time, y, x) float64 dask.array<chunksize=(2, 1100, 4096), meta=np.ndarray>
    B02             (time, y, x) float64 dask.array<chunksize=(3, 1100, 4096), meta=np.ndarray>
```

Since users can get the `start_time` and `end_time` of each band from the attributes, could we change the time dimension of time_bounds to `time_bnds`? Otherwise, it's quite hard to determine the `start_time` and `end_time` attributes are caused by different time periods or different channel with slightly different time.

Note that, in my opinion, `time_bounds` should be the minimum and maximum time at each time periods. So, the slightly different time of different band should obey the rule.